css/notes.css

```diff
@@ -9,7 +9,6 @@
 }
 
 #app-navigation #note-add span {
-    display: none;
```

Comment:
I vote for displaying the text always. And I vote for displaying it more like a button, like in the contacts app (cp. https://github.com/nextcloud/notes/issues/14#issuecomment-256340853):
![neuer-kontakt](https://cloud.githubusercontent.com/assets/6277619/19726700/745d175c-9b8c-11e6-8f78-d12336c9b7ee.jpg)